apps/web/src/components/TrendingSuggestionList/TrendingSuggestionList.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { useQuery } from 'react-query';
+import TrendingItemSuggestion from '../TrendingItemSuggestion/TrendingItemSuggestion';
+import './TrendingSuggestionList.scss';
+
+interface Item {
+  /** How we identify each item */
+  id: number;
+  /** How can we view the suggested post? */
+  URL: string;
+  /** Suggestion Title */
+  trendingSuggestionHeading: string;
+  /** Suggestion Summary */
+  trendingSuggestionSummary: string;
+  /** Specific forum the suggestion lives in */
+  targetSubreddit: {
+    /** Forum's Icon */
+    subRedditIcon: React.ReactElement;
+    /** Forum's title */
+    subRedditName: string;
+  };
+}
+
+interface Props {
+  customTestId?: string;
+}
+
+const TrendingSuggestionList = ({
+  customTestId = 'suggestion-list',
+}: Props): React.ReactElement => {
+  const { isLoading, data, isFetching, isFetched } = useQuery(
+    'suggested-items',
+    async () => {
+      const response = await fetch('../../Data/SuggestedItems.json');
```

Comment:
I like the way you did this, but we should switch to a method that is closer to the actual user experience, so something like Mock Service Worker. 

https://mswjs.io/

I'll make a story for adding MSW to the project.